tripal_analysis_expression/includes/TripalFields/local__expression_features/local__expression_features.inc

```diff
@@ -0,0 +1,190 @@
+  // If you override this variable in a child class be sure to replicate the
+  // term_name, term_vocab, term_accession and term_fixed keys as these are
+  // required for all TripalFields.
+  public static $default_instance_settings = array(
+    // The DATABASE name, as it appears in chado.db.  This also builds the link-out url.  In most cases this will simply be the CV name.  In some cases (EDAM) this will be the SUBONTOLOGY.
+    'term_vocabulary' => 'local',
+    // The name of the term.
+    'term_name' => 'expression_features',
+    // The unique ID (i.e. accession) of the term.
+    'term_accession' => '12345',
+    // Set to TRUE if the site admin is not allowed to change the term
+    // type, otherwise the admin can change the term mapped to a field.
+    'term_fixed' => FALSE,
+    // Indicates if this field should be automatically attached to display
+    // or web services or if this field should be loaded separately. This
+    // is convenient for speed.  Fields that are slow should for loading
+    // should have auto_attach set to FALSE so tha their values can be
+    // attached asynchronously.
+    'auto_attach' => FALSE,
+    // The table in Chado that the instance maps to.
+    'chado_table' => '',
+    // The column of the table in Chado where the value of the field comes from.
+    'chado_column' => '',
+    // The base table.
+    'base_table' => '',
+  );
+
+  // A boolean specifying that users should not be allowed to create
+  // fields and instances of this field type through the UI. Such
+  // fields can only be created programmatically with field_create_field()
+  // and field_create_instance().
+  public static $no_ui = FALSE;
+
+  // A boolean specifying that the field will not contain any data. This
+  // should exclude the field from web services or downloads.  An example
+  // could be a quick search field that appears on the page that redirects
+  // the user but otherwise provides no data.
+  public static $no_data = FALSE;
+
+  /**
+   * @see ChadoField::load()
+   *
+   **/
+
+  public function load($entity)
+  {
+
+    // ChadoFields automatically load the chado column specified in the
+    // default settings above. If that is all you need then you don't even
+    // need to implement this function. However, if you need to add any
+    // additional data to be used in the display, you should add it here.
+    $pvalue_cvterm_id = chado_get_cvterm([
+      'cv_id' => ['name' => 'OBI'],
+      'name' => 'p-value',
+    ])->cvterm_id;
+    $evidence_code_cvterm_id = chado_get_cvterm([
+      'cv_id' => ['name' => 'local'],
+      'name' => 'evidence code',
+    ])->cvterm_id;
+    $header = ['Feature Name', 'Keyword', 'Relationship', 'P-value'];
+    $rows = [];
+    $query = chado_db_select('analysis', 'a');
+    #$analysis_id = $entity->chado_record->analysis_id;
+    $query->fields('f', array('name')); //feature name and id for link
+    #$query->condition('analysis_id', $analysis_id, '=');
```

Comment:
You need this condition to make sure that the features are associated with the analysis that's being viewed currently.